.github/workflows/NodeCI.yml

```diff
@@ -46,12 +44,12 @@ jobs:
         node-version: ${{ matrix.node-version }}
     - name: Install @typescript-eslint v4
       run: |+
-        npm i -D @typescript-eslint/parser@4 @typescript-eslint/eslint-plugin@4 --legacy-peer-deps
-        npx rimraf node_modules
+        yarn add -D @typescript-eslint/parser@4 @typescript-eslint/eslint-plugin@4 --ignore-engines
+        rm -rf node_modules
```

Comment:
Is removing `node_modules` required? I don't see its benefit, actually.

Reply:
Previously It was sometimes confusing with `npm install` and couldn't be installed successfully. Therefore, it tries to keep it clean by deleting node_modules once.
We replaced it with yarn so we may not need it anymore.

Reply:
We can give it a try, deleting `node_modules` and reinstall may increase the CI running time.